profiles/GurjeetSinghVirdee.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/73753957/147848925-4655ce63-2950-4deb-9931-e8140909bf74.png",
+  "issueId": 950,
+  "name": "Gurjeet Singh Virdee",
+  "username": "gurjeetsinghvirdee"
+}
```

```suggestion
}

```

Add newline after line number `6`.